src/test/resources/features/highlighting/inspections/redundant_coma_inspection.feature

```diff
@@ -41,4 +41,54 @@ Feature: Inspection for redundant coma in the end of the line
       | rule_like  | section |
       | rule       | input   |
       | rule       | params  |
-      | checkpoint | input   |
\ No newline at end of file
+      | checkpoint | input   |
+
+  Scenario Outline: Fix redundant coma in the end of the line with two parameters
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    <rule_like> NAME:
+      <section>: "input1.txt", "input2.txt",
+    """
+    And SmkRedundantComaInspection inspection is enabled
+    Then I expect inspection warning on <,> in <<section>: "input1.txt", "input2.txt",> with message
+    """
+    Coma is unnecessary
+    """
+    When I check highlighting warnings
+    Then I invoke quick fix Remove redundant coma and see text:
+     """
+    <rule_like> NAME:
+      <section>: "input1.txt", "input2.txt"
+    """
+    Examples:
+      | rule_like  | section |
+      | rule       | input   |
+      | rule       | params  |
+      | checkpoint | input   |
+
+  Scenario: Check no inspection in python code in dict line
+    Given a snakemake project
+    Given I open a file "foo.smk" with text
+    """
+    dict(a=0,
+         b=1,
+         c=2,
+         )
+    """
+    And SmkRedundantComaInspection inspection is enabled
+    Then I expect no inspection warning
```

Comment:
`When I check highlighting warnings` step is missing, step checks nothing.

Reply:
Done